src/assets/labContent.js

```diff
@@ -161,16 +161,16 @@ export default [
       subheader: "How deep do the effect of advertisements run in society itself?",
       bodyText: [
         {
-          body: <p>Here is a graph displaying the <mark className="bold">gender distribution of computer scientists from the 1960s to now.</mark></p>
+          body: <p>Here is a graph displaying the <mark className="bold">percentage of women enrolled in computer science degrees from the 1960s to now.</mark> (<a href='https://www.npr.org/sections/money/2014/10/21/357629765/when-women-stopped-coding' target="_blank" rel="noopener noreferrer">source</a>)</p>
```

Comment:
Instead of just saying hyperlinking the source to the word source, do you want to put NPR or npr.org to specifically say where it's from within the hyperlink itself?